stubs/fpdf2/fpdf/fpdf.pyi

```diff
@@ -288,8 +300,12 @@ class FPDF:
     def set_font_size(self, size: float) -> None: ...
     def set_char_spacing(self, spacing: float) -> None: ...
     def set_stretching(self, stretching: float) -> None: ...
-    def add_link(self) -> int: ...
-    def set_link(self, link, y: int = ..., x: int = ..., page: int = ..., zoom: float | Literal["null"] = ...) -> None: ...
+    def add_link(
+        self, y: float = 0, x: float = 0, page: int = -1, zoom: float | Literal["null"] = "null"  # noqa: Y020
```

Comment:
Can you file this as a bug over at flake8-pyi so we track it?

Reply:
Oh wait it's already been fixed, hasn't it

Reply:
Don't mind me

Reply:
Thanks! I forgot that I merged https://github.com/PyCQA/flake8-pyi/pull/318/ only last week, which I think already fixed this